<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jogo da Velha</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images.jpg</normaloff>images.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_32">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_33">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Cor_Tabuleiro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_Cor_Tabuleiro</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton_21</zorder>
   <zorder>pushButton_31</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_22</zorder>
   <zorder>pushButton_32</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_23</zorder>
   <zorder>pushButton_33</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionJogador_Inicia"/>
    <addaction name="separator"/>
    <addaction name="actionIniciar_Jogo"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionQT"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionJogador_Inicia">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Jogador Inicia</string>
   </property>
  </action>
  <action name="actionIniciar_Jogo">
   <property name="text">
    <string>Iniciar Jogo</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre ...</string>
   </property>
  </action>
  <action name="actionQT">
   <property name="text">
    <string>Sobre o QT</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
